### ISSUE TITLE
Optimise lightbox for different viewports

### ISSUE THREAD

**Issue by photoMaldives** [photopress-dev/photopress-plugin#5]
Gave this plugin a spin, and it shows lots of potential.
However, one showstopper for me (and my theme) was the lightbox (without customisation) - 
1) vertical images exceeded the viewport, requiring up/down scrolling;
2) horizontal images were displayed a little too small (and smaller than their actual size).
I'm unsure if this was down to pad/margins of my theme, or the vh/vw settings of the lightbox.
Thanks, and keep up the good work.

**Comment by padams** [comment]
Any chance you could provide a URL so we can debug?

**Comment by photoMaldives** [comment]
Not so easily, as I'm still developing.
However, I just installed and setup PhotoPress on another theme (Astra) ... and it works very well!  :-) It's fast, no frills, and all 4 of the gallery styles work well. The lightbox fits nicely to viewport size.
So likely my initial impressions were completely down to my theme.
Two **very minor** things in my new setup, with the lightbox, when caption is on right:
1) Portrait/vertical images seem to be positioned relative to the size of the caption, causing their position to change slightly when moving from one to the next (for example, if one photo has a small caption and the subsequent photo has a much longer caption).
2) Landscape/horizontal images don't seem to have this issue, however the caption can overlap the right scroll arrow (suggest: move caption to bottom right? Or have a user toggle: top/bottom).
_Bit neither of these are an issue when selecting caption at bottom._
Thanks. And I will now go back to WPT to comment accordingly. Many apologies of I came over as overly-critical.
